src/log.py

```diff
@@ -0,0 +1,94 @@
+"""
+log
+
+By: Norma Chang
+"""
+import json
+import time
+import os.path
+
+
+class Log:
+    """
+    Logging class.
+
+    Logs data from the properties of an instance of an vehicle class for
+    testing purposes.
+    """
+    def __init__(self):
+
+        self.file_name = ""
+        self.class_name = ""
+        self._start_time = time.strftime('%b %d %Y %H:%M:%S', time.gmtime())
+        self._start_time = self._start_time.replace(" ", "").replace(":", "")
+
+
+    def isFollower(self, vehicle):
+        if vehicle.__class__.__name__ == 'Follower':
```

Comment:
```python
return vehicle.__class__.__name__ == 'Follower'
```

is the exact same thing and it's cleaner. I'm not sure if this is the best way to do this, but I guess it works for now.